src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeEach
+    void userStorageInitialization() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User(1L,"valid_login", "123456", 40);
+        Storage.people.add(new User(2L,"LarsUrlich","myolddrums",60));
+    }
+
+    @AfterEach
+    void clearStorage() {
+        Storage.people.clear();
+    }
+
+    @Test

```

Comment:
Try to group tests for the same logic together = so all tests for 'age' to be one after another, for better readability. It will be seend which tests are missing. For example for login a positive test case is missing (login ok)
Pls format the code: Ctrl+Alt+L (Windows/Linux) or⌥⌘L (macOS)

Reply:
ok, done